packages/block-library/src/latest-comments/editor.scss

```diff
@@ -0,0 +1,17 @@
+// Higher specificity - target list via wrapper.
+.wp-block-latest-comments .wp-block-latest-comments {
+	// Remove spacing. Higher specificity required to
+	// override default wp-block layout styles in the Post/Site editor.
+	padding: 0;
```

Comment:
I would prefer applying the `padding: 0` over changing the wrapper, because it is a smaller change.
And the markup in the editor is already pretty complex.

Reply:
For clarity, my suggestion was only to swap the wrapper from this block's server-side rendering to a list element instead of a div. Then apply any necessary reset styles. I don't think that makes the markup in the editor more complex.

The main goal for me is visual consistency for the user between the editor and frontend. I ultimately don't have a strong opinion on whether that's done by tweaking the block's editor rendering or the additional padding reset on the frontend. I am a bit wary of anything that could be a breaking change for frontend styles though.

As you note it is a small change so perhaps it is something that could be merged early in the release cycle and called out?